src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -180,7 +187,16 @@ public function normalize($object, string $format = null, array $context = [])
             }
 
             $attributeContext = $this->getAttributeNormalizationContext($object, $attribute, $context);
-            $attributeValue = $this->getAttributeValue($object, $attribute, $format, $attributeContext);
+
+            try {
+                $attributeValue = $this->getAttributeValue($object, $attribute, $format, $attributeContext);
+            } catch (UninitializedPropertyException $exception) {
```

Comment:
let's rename this to `$e` for consistency

Reply:
Done